File: RagnarokBotWeb/Domain/Services/Dto/PaymentDto.cs


using RagnarokBotWeb.Domain.Enums;

namespace RagnarokBotWeb.Domain.Services.Dto
{
    public class PaymentDto
    {
        public long Id { get; set; }
        public SubscriptionDto Subscription { get; set; }
        public DateTime? ConfirmDate { get; set; }
        public EPaymentStatus Status { get; set; }
        public DateTime? ExpireAt { get; set; }
        public bool IsExpired { get; set; }
        public string? Url { get; set; }

    }
}

File: RagnarokBotWeb/ClientApp/src/app/models/payment.dto.ts
import { SubscriptionDto } from "./subscription.dto";

export class PaymentDto {
    id: number;
    confirmDate?: Date;
    status: any;
    subscription: SubscriptionDto;
    url?: string;
    isExpired: boolean;
    orderNumber?: string;
    expireAt: Date;
}
